src/System.ValueTuple/src/System/ValueTuple/ValueTuple.cs

```diff
@@ -405,7 +411,12 @@ int ITupleInternal.GetHashCode(IEqualityComparer comparer)
         /// </remarks>
         public override string ToString()
         {
-            return "(" + Item1 + ")";
+            return "(" + Item1.ToString() + ")";
```

Comment:
> string interpolation over string concatenation

Why?

> // no need to call ToString()

There's no "need" to call ToString() with the concat version, either, but it's good for performance to do so.  https://github.com/dotnet/corefx/pull/8025#discussion-diff-60845458